rom-rails.gemspec

```diff
@@ -23,10 +23,9 @@ Gem::Specification.new do |spec|
   }
 
   spec.add_runtime_dependency 'addressable', '~> 2.3'
-  spec.add_runtime_dependency 'dry-core', '~> 0.4'
-  spec.add_runtime_dependency 'dry-equalizer', '~> 0.2'
-  spec.add_runtime_dependency 'railties', '>= 3.0', '< 6.2'
-  spec.add_runtime_dependency 'rom', '~> 5.2'
+  spec.add_runtime_dependency 'dry-core', '~> 1.0'
+  spec.add_runtime_dependency 'railties', '>= 3.0', '<= 7.0.4'
```

Comment:
Maybe `'< 7.0.5'` insteaf of `'<= 7.0.4'`? It will include versions like  `7.0.4.3`.

Reply:
Why not just do `'< 8'` to allow for any version of Rails 7?

Reply:
Because Rails doesn't acutally follow SemVer, and there's no garuntee that `7.1` doesn't break things.

that said, This should probably be `<= 7.1.0` and not get too hung up on the patch version.